docs/console/05-01-console.md

```diff
@@ -20,3 +20,4 @@ This table lists the configurable parameters, their descriptions, and default va
 | **cluster.headerLogoUrl** | Defines the address of the logo image that shows in the Console UI navigation header. | `assets/logo.svg` |
 | **cluster.faviconUrl** | Defines the icon that shows in the address bar of a browser. | `favicon.ico` |
 | **cluster.headerTitle** | Defines an additional title that shows next to the logo image in the Console UI navigation header. | None |
+| **cluster.disabledNavigationNodes** | An array of categories or specific nodes that you want to hide in the Console UI navigation. To hide a category, make sure the array includes **categoryLabel** for cluster-wide views. For the namespace-related views, add the **namespace** prefix to the label: **namespace.categoryLabel**. To hide a specific node, make sure the array includes **categoryLabel.nodeLabel** or **namespace.categoryLabel.nodeLabel**. For all labels, use lowercase and don't use any spaces or dashes. | [] |
```

Comment:
```suggestion
| **cluster.disabledNavigationNodes** | An array of categories or specific nodes that you want to hide in the Console UI navigation. To hide a category, make sure the array includes **categoryLabel** for cluster-wide views. For the Namespace-related views, add the **namespace** prefix to the **namespace.categoryLabel** label. To hide a specific node, make sure the array includes **categoryLabel.nodeLabel** or **namespace.categoryLabel.nodeLabel**. For all labels, use lowercase and don't use any spaces or dashes. | [] |
```

Reply:
And can we start the description with a verb? what does `[]` denote? We normally put values in code here.

Reply:
I guess `[]` stands for an array? But I agree with the verb part of the comment 👍